python/batch-predictor/docker/main.py

```diff
@@ -0,0 +1,3 @@
+from merlinpyspark.__main__ import main
+
+main()
```

Comment:
This file is a simple wrapper to invoke merlinpyspark.main() function, because SparkApplication need an explicit path to the main application file (config ref: https://github.com/caraml-dev/helm-charts/blob/main/charts/merlin/templates/_config.tpl#L29)